cordova-lib/spec-cordova/HooksRunner.spec.js

```diff
@@ -249,7 +249,7 @@ describe('HooksRunner', function() {
                 var hooksOrderFile = path.join(projectRoot, 'hooks_order.txt');
 
                 return hooksRunner.fire(test_event, hookOptions).then(function() {
-                    expect(hooksOrderFile).toExist();
+                    expect(hooksOrderFile).toBeDefined();
```

Comment:
toExist and toBeDefined are pretty different. Don't think this change is the way to go.